flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -322,6 +322,19 @@ public Builder setSnapshotProperty(String property, String value) {
      */
     @Deprecated
     public DataStreamSink<RowData> build() {
```

Comment:
`build` can forward the execution to `append`.

Reply:
done

Reply:
Since this returns the same actual object, it should be fine. How about just adding an unchecked cast with a suppression?